### ISSUE TITLE
Inline lambda in argument list causes problems for argument chopping and lambda alignment

### ISSUE THREAD

**Issue by cmeeren** [fsprojects/fantomas#1028]
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKBbBBXANosAMgJYBmi0AntHmALwA6ATvQHZOvssR4AuYRLInzgsSRAOZ1OECAEMAzhRbQwwaTIjQSkgDwBaMCLHj1G%2BLACEpjWACyC%2BQBVGslvKEA6AUOs3D2jwJYcXFERgAxWWgeWEYKf3EPAEEALyxGRABlMIA3ImhEACEseQBhWBYWTJ5GBI8ARSxEJoA5WXR8LUSgkLDI6NiKXxsAChIsLgzJPQA%2BYb8pj3K3LA6wAFV5MI8AC1d4PE2wgElBHiJZHkR4ACVEcSJ5GsuiCvmbReX5Vfwjxl39octox1gAHeCXa7vDSfCrfNZ-AEsA6IADiiB42Xk7gqf2Ot0QAEcmk9oWAAJTWAC%2BkAANOAoFseOcWOJ5KgwABtBkQUB%2BSAAEh4FFByDQkAEPDpPJkgpyshwTQ53L8EAADPTVVBTkgWJiiClkJrVQAmGUQAC6Mqpxt55qgQpFYqgkul-IF8sVzpVqoAjLabJB7AAPYgsRAERCsng7N2qgAcavtVo0NplfMDjtFHMgACNYLAcHGNHKFUq0D6-CaAyXsuh8oWKkkeABRZEAeRIYaN9ogJAVW2T1oDGZLWedeYLRZrDs95a5vbAAGYZxBIJlQVEiogSLFEAAFWQuDpXRjFmw1JpD1Mj%2B2C4XZ8X5wvntcesvexcAFlXUA3W8KHc9yCAB3MJoAURBThyWAIPOCpXxkS9EGvGQ0w0UdZXHHNn2nO93y9ZVFwAVl-ddNwKQDdwyMFRUYCCtmg2CXgQ39%2BxwQc-BTNDb3dB8J1wxDS0IitFwANjI-9KKAjJShwBwvkvAYzzYgcUK44d03w-icKnISCPnSsbAAdkkijt2oxBbEQdBczCRC%2BzU1CIHQmRMLfHSnz0mdhMMxd4zMgCZMQcocFYxd2M4mxuJc3jM08qBBJ8gzPy1ABOQKCiSEhT3KdB0FkBywGQ5ywFcu0%2BKdXSX2SudUr9DVF3IoLLLrBswpYIrIvU6LNIw7Sqq8mr8LqoitV9f0mqkxBstPNq4A6oqSo0m8tMqx9Eu8kaPzGv1qyanUox4dsWGcCgAHUhFjVSOJ6jQYrKuKxwSydhvdUbRPGlcpvMpJGFgCZ4AAEUQHAiHrU8lsYK8Vp4tb4sGl09X0j6F3Gn8mpDY4SEcHYoxbW7Mh2WIeEu%2BAYyKr8k1h2L4eexGJWR2qds%2Bv1SMx2Rg1OMRg3bOjLliFtg1BDJsVeTrf2I6netW-r1onV1mZEtG-QkjngzuOBGHgMmKd-KnSvK1QBo2xmpSVvzxtM9W-pcCh20YYgnl167v2l%2B6%2Brck2FaZ7blaMjRfQC9WADUP0KAR4AEcQXcp92ZAeo33IdF7Fb9y2-Qy9XwgmaJxYj5Fo9j-X48tT2KoR02kvT%2Bqq0arUg1wc4wfDQowugABrQoXE7jF5BOzJ2hCwtVgliKnJpx66awl7q-elmVarSaG5aRAQJb7ceDAqNHCdEGxDOcWkmRazbLCdkbqij3Za9%2BXqrw%2Bf-cXE19obgBpRBEFBbHcajU5B46N2LqE8ZZwzlpXH25sa67SrN9BuIYCb1nkDsYubtDZPRngzOemZUYBxkCaDGDdMjRzwH9cQPw9TnUQLmWwCAexam6ug6eHksFbUfhnKs7MG5JDBuIFgOdlDwSqBIFglx0iIEcLAQ6eoWJjwYSA6%2BYDb4QPvijBeeCIAmjVtwnAp5RHnByBGCo4hT52UYPvbwQiL7j1ukw8B9Mq5sJweo5%2B1siE1HyDwWhSBgG2MnlSBkFpjQSnkOEdwHJGFUiAA)

#### Code

```fsharp
module Lifecycle =


  let init config =
    async {
      cfg <- config
      do!
        MassTransit.init
          cfg.LoggerFactory cfg.AzureServiceBusConnStr cfg.QueueName cfg.LoggerFactory
          (fun reg ->
            reg.Consume User.handleUserInitiatedRegistration
            reg.Consume User.handleUserUpdated
            reg.Consume User.handleGetSessionUserIdRequest
          )
    }
```
            

#### Result

```fsharp
module Lifecycle =


  let init config =
    async {
      cfg <- config
      do! MassTransit.init cfg.LoggerFactory cfg.AzureServiceBusConnStr
            cfg.QueueName cfg.LoggerFactory (fun reg ->
            reg.Consume User.handleUserInitiatedRegistration
            reg.Consume User.handleUserUpdated
            reg.Consume User.handleGetSessionUserIdRequest)
    }

```
            
#### Problem description

Due to the inline lambda, the arguments don't break correctly. I would expect:

```fsharp
module Lifecycle =


  let init config =
    async {
      cfg <- config
      do! MassTransit.init
            cfg.LoggerFactory
            cfg.AzureServiceBusConnStr
            cfg.QueueName
            cfg.LoggerFactory
            (fun reg ->
               reg.Consume User.handleUserInitiatedRegistration
               reg.Consume User.handleUserUpdated
               reg.Consume User.handleGetSessionUserIdRequest)
    }
```

Or something similar.

#### Extra information

- [ ] The formatted result breaks by code.
- [ ] The formatted result gives compiler warnings.
- [ ] I or my company would be willing to help fix this. *(I'll help in any way I can except implementation; I don't have the capacity to ramp up on a new codebase now)*

#### Options

Fantomas Master at 08/21/2020 08:12:02 - 866535662e4cc286f84e893ef7ff5cc5cabae31c

```fsharp
    { config with
                IndentSize = 2
                MaxLineLength = 80 }
```

**Comment by cmeeren** [comment]
Judging from fantomas-online, this seems to be fixed in 4.3.

**Comment by nojaf** [comment]
Great, I'll add a regression test someday.
